### ISSUE TITLE
Example thoroughly broken

### ISSUE THREAD

**Issue by neil-s** [driftyco/ionic-ion-tinder-cards#60]
I'm trying to run the example code provided, but it seems to be outdated and broken. It relies on files that don't exist, including swipecards.js. And when I try to fix the code and run it, I still get an empty cards array for some reason. Not sure how to fix this, or I would have updated it and submitted a pull request myself.

**Comment by keithdmoore** [comment]
Agreed.  The example needs some love.  Until someone has time to fix it, maybe take a look at this example:  https://github.com/trendicity/trendicity

**Comment by neil-s** [comment]
Thanks @keithdmoore for the quick response. I have very little experience with Angular, so that example is quite overwhelming in its complexity, hard to see where in the Javascript for the example in this repo do the card objects need to be instantiated.

**Comment by keithdmoore** [comment]
I see.  The cards in trendicity are actually in the form of instagram posts.  They get initialized in  `home.js` like this:  
```
$scope.model =  PostsService.getModel();
```

**Comment by danieljin** [comment]
+1. Thanks for this. Wish the README was more descriptive.
